src/frequenz/channels/util/_file_watcher.py

```diff
@@ -5,7 +5,7 @@
 import asyncio
 import pathlib
 from enum import Enum
-from typing import List, Optional, Set, Union
+from typing import List, Optional, Set, Tuple, Union
```

Comment:
When you update old code to add more built-in types it would be nice if you convert it to using `from __future__ import annotations` instead. It is optional but very desirable so we can slowly get rid of the old idioms.

Reply:
The Tuple will no longer be needed as we will use the `Event` class, however I'll add a commit to use annotations from __future__  instead